sdk/tests/conformance2/extensions/webgl-multi-draw-instanced-base-vertex-base-instance.html

```diff
@@ -469,8 +469,8 @@
     gl.disable(gl.SCISSOR_TEST);
     gl.clearBufferfv(gl.COLOR, 0, [1,0,0,1]);
 
-    const last_gl_vert_id = desc.base_vert + desc.vert_count - 1;
-    const last_vert_id = desc.first_vert + last_gl_vert_id;
+    const last_gl_vert_id = desc.first_vert + desc.base_vert + desc.vert_count - 1;
```

Comment:
`desc.base_vert + desc.first_vert`

Reply:
vert_count can be 0, 1, 2 so they are not equal?

Reply:
Oh I think you mean flip the order...
```diff
- const last_gl_vert_id = desc.first_vert + desc.base_vert + desc.vert_count - 1;
+ const last_gl_vert_id = desc.base_vert + desc.first_vert + desc.vert_count - 1;
```